packages/easy-coding-standard/src/Config/ECSConfig.php

```diff
@@ -21,74 +21,86 @@ final class ECSConfig extends ContainerConfigurator
     /**
      * @param string[] $paths
      */
-    public function paths(array $paths): void
+    public function paths(array $paths): self
```

Comment:
We don't use fluent on purpose, as the return types might collide and create ambiguous API.

See https://ocramius.github.io/blog/fluent-interfaces-are-evil